types/index.d.ts

```diff
@@ -261,9 +261,10 @@ declare namespace Realm {
         set(element:{[key:string]: ValueType}): void;
 
```

Comment:
Is `set` also returning a Dictionary?

Reply:
Good catch. Yes, it does.

Reply:
Cool.  The comment also needs to be updated